tv-app/effecticeTV/app/src/main/java/band/effective/office/tv/network/leader/LeaderApi.kt

```diff
@@ -1,13 +1,14 @@
-package band.effective.office.tv.leader
+package band.effective.office.tv.network.leader
 
+import band.effective.office.tv.network.leader.models.Response
 import retrofit2.Call
 import retrofit2.http.GET
 import retrofit2.http.Query
 
 interface LeaderApi {
     @GET("api/v4/events/search")
-    fun searchEvents(@Query("paginationSize") count: Int, @Query("placeIds[]") placeId: Int): Call<SearchResponse>
+    fun searchEvents(@Query("paginationSize") count: Int, @Query("placeIds[]") placeId: Int): Call<Response>
     @GET("api/v4/events/search")
-    fun searchEventsByCity(@Query("paginationSize") count: Int, @Query("cityId") cityId: Int): Call<SearchResponse>
+    fun searchEventsByCity(@Query("paginationSize") count: Int, @Query("cityId") cityId: Int): Call<Response>
```

Comment:
There are two functions for the same endpoint is not good I think. 

Why not just add all the parameters to one function above?

Reply:
Yes, but this logic can be moved from api service like this:

For example: 
1. Define these methods on Repository level
2. Each function call only one `searchEvents` from `LeaderApi` but provide different parameters set